compilation_tests/ctest_trivially_copyable.cpp

```diff
@@ -167,46 +167,30 @@ MDSPAN_STATIC_TEST(
 
 MDSPAN_STATIC_TEST(
   std::is_trivially_copyable<
-    stdex::layout_stride<stdex::dynamic_extent, 1, 3>::template mapping<
+    stdex::layout_stride::template mapping<
       stdex::extents<42, stdex::dynamic_extent, 73>
     >
   >::value
 );
 
 MDSPAN_STATIC_TEST(
   std::is_trivially_copyable<
-    stdex::layout_stride<stdex::dynamic_extent, 1, 3>::template mapping<
+    stdex::layout_stride::template mapping<
       stdex::extents<42, 27, 73>
     >
   >::value
 );
 
 MDSPAN_STATIC_TEST(
   std::is_trivially_copyable<
-    stdex::layout_stride<12, 1, 3>::template mapping<
```

Comment:
now duplicated tests are removed, just maintain the ones with were all dynamic extent parameters.